### PR TITLE
debug bitrise builds

### PR DESCRIPTION
**Pull request by bluwave**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/0eqtftg) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F0eqtftg)
teacher: [Link](https://rebrand.ly/08h20v0) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F08h20v0)
parent: [Link](https://rebrand.ly/y2oueca) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fy2oueca)

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026017&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a release note. If no release note is wanted, use `none`. Example: `release note: Fixed a bug that prevented users from enjoying the app.`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add which apps this change affects. Example: `affects: Teacher, Student` or `affects: none`
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Please add a reference to a jira ticket. For example: `refs: MBL-10023`
  </td>
    </tr>
  </tbody>
</table>



Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.63% | 93.63% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026047&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026049&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift [hunk 1]
```diff
@@ -304,6 +307,9 @@ extension AssignmentListViewController: UITableViewDataSource, UITableViewDelega
     }
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let arr = ["a", "b"]
+        let value = arr[3]
+        print(value)
```
Comment (bluwave) [inline Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift]: @armstrongnate - here

File: Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift [hunk 2]
```diff
@@ -206,6 +208,7 @@ class AssignmentListViewController: UIViewController, ColoredNavViewProtocol, Er
     }
 
     @IBAction func actionFilterClicked(_ sender: UIButton) {
+        Crashlytics.sharedInstance().crash()
```
Comment (bluwave) [inline Student/Student/Assignments/AssignmentList/AssignmentListViewController.swift]: @armstrongnate here

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026052&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026053&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1026171&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Pull request closed without merging**